packages/cctp-plugin/src/index.ts

```diff
@@ -0,0 +1,88 @@
+import { createPlugin } from "every-plugin";
+import { Effect } from "every-plugin/effect";
+import { z } from "every-plugin/zod";
+
+import { contract } from "./contract";
+import { CCTPService } from "./cctp-service";
+
+/**
+ * CCTP Data Provider Plugin - Circle's Cross-Chain Transfer Protocol
+ * 
+ * Collects bridge metrics from Circle's CCTP including:
+ * - Volume data from Celer cBridge API
+ * - Rate quotes and fees from CCTP API
+ * - Liquidity depth from fast burn allowance
+ * - Supported USDC assets across all chains
+ * 
+ * Note: CCTP public APIs require NO authentication
+ */
+export default createPlugin({
+  id: "@nearbuilders/cctp",
+
+  variables: z.object({
+    baseUrl: z.string().url().default("https://iris-api.circle.com"),
+    timeout: z.number().min(1000).max(60000).default(10000),
+    maxRetries: z.number().min(1).max(10).default(3),
+    initialBackoffMs: z.number().min(100).max(5000).default(1000),
+  }),
+
+  secrets: z.object({
+    // CCTP APIs are public - no API key needed
+    // This is kept for contract compatibility but can be empty
+    apiKey: z.string().optional().default(""),
```

Comment:
You can remove this, make it an empty object

Remove any variables or secrets that are unused.